relational_operators/CreateIndexOperator.hpp

```diff
@@ -75,6 +84,8 @@ class CreateIndexOperator : public RelationalOperator {
  private:
   CatalogRelation *relation_;
   const std::string &index_name_;
+  const std::vector<IndexSubBlockDescription> index_descriptions_;
```

Comment:
I see, then we may want to use the rvalue via `std::move` in the constructor to avoid a deep copy.